{"uid":"56725564af960f94","name":"Exception test","fullName":"tests.forgot_password_test.TestForgotPassword#test_expected_exception_on_page_title","historyId":"288cd0030c92ba34b4ff4b58ed8ec476","time":{"start":1714094081908,"stop":1714094092251,"duration":10343},"description":"Exception catching","descriptionHtml":"<p>Exception catching</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560343153e33 <unknown>\n#1 0x560342e43567 <unknown>\n#2 0x560342e8e27d <unknown>\n#3 0x560342e8e331 <unknown>\n#4 0x560342ed1624 <unknown>\n#5 0x560342eb038d <unknown>\n#6 0x560342eceae2 <unknown>\n#7 0x560342eb0103 <unknown>\n#8 0x560342e811ba <unknown>\n#9 0x560342e81b8e <unknown>\n#10 0x56034311859b <unknown>\n#11 0x56034311c34b <unknown>\n#12 0x560343104681 <unknown>\n#13 0x56034311ceb2 <unknown>\n#14 0x5603430e96af <unknown>\n#15 0x560343143428 <unknown>\n#16 0x560343143600 <unknown>\n#17 0x560343152f84 <unknown>\n#18 0x7f2b6d494ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7fe5541670e0>\n\n    @allure.description(\"Exception catching\")\n    @allure.title(\"Exception test\")\n    @allure.link(\"github.com/allure-examples/\", name=\"Allure Examples\")\n    @allure.issue(\"github.com/allure-examples/allure-examples/issues/1\", name=\"ISSUE-1\")\n    @allure.testcase(\n        \"github.com/allure-examples/allure-examples/issues/2\", name=\"TESTCASE-2\"\n    )\n    def test_expected_exception_on_page_title(self):\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"72836af0885a2fea762252971893f37f\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fe553e9bba0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x560343153e33 <unknown>\nE       #1 0x560342e43567 <unknown>\nE       #2 0x560342e8e27d <unknown>\nE       #3 0x560342e8e331 <unknown>\nE       #4 0x560342ed1624 <unknown>\nE       #5 0x560342eb038d <unknown>\nE       #6 0x560342eceae2 <unknown>\nE       #7 0x560342eb0103 <unknown>\nE       #8 0x560342e811ba <unknown>\nE       #9 0x560342e81b8e <unknown>\nE       #10 0x56034311859b <unknown>\nE       #11 0x56034311c34b <unknown>\nE       #12 0x560343104681 <unknown>\nE       #13 0x56034311ceb2 <unknown>\nE       #14 0x5603430e96af <unknown>\nE       #15 0x560343143428 <unknown>\nE       #16 0x560343143600 <unknown>\nE       #17 0x560343152f84 <unknown>\nE       #18 0x7f2b6d494ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1714093862218,"stop":1714093862218,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1714093862218,"stop":1714093862218,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1714093862219,"stop":1714093862219,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Exception catching","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560343153e33 <unknown>\n#1 0x560342e43567 <unknown>\n#2 0x560342e8e27d <unknown>\n#3 0x560342e8e331 <unknown>\n#4 0x560342ed1624 <unknown>\n#5 0x560342eb038d <unknown>\n#6 0x560342eceae2 <unknown>\n#7 0x560342eb0103 <unknown>\n#8 0x560342e811ba <unknown>\n#9 0x560342e81b8e <unknown>\n#10 0x56034311859b <unknown>\n#11 0x56034311c34b <unknown>\n#12 0x560343104681 <unknown>\n#13 0x56034311ceb2 <unknown>\n#14 0x5603430e96af <unknown>\n#15 0x560343143428 <unknown>\n#16 0x560343143600 <unknown>\n#17 0x560343152f84 <unknown>\n#18 0x7f2b6d494ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7fe5541670e0>\n\n    @allure.description(\"Exception catching\")\n    @allure.title(\"Exception test\")\n    @allure.link(\"github.com/allure-examples/\", name=\"Allure Examples\")\n    @allure.issue(\"github.com/allure-examples/allure-examples/issues/1\", name=\"ISSUE-1\")\n    @allure.testcase(\n        \"github.com/allure-examples/allure-examples/issues/2\", name=\"TESTCASE-2\"\n    )\n    def test_expected_exception_on_page_title(self):\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"72836af0885a2fea762252971893f37f\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fe553e9bba0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x560343153e33 <unknown>\nE       #1 0x560342e43567 <unknown>\nE       #2 0x560342e8e27d <unknown>\nE       #3 0x560342e8e331 <unknown>\nE       #4 0x560342ed1624 <unknown>\nE       #5 0x560342eb038d <unknown>\nE       #6 0x560342eceae2 <unknown>\nE       #7 0x560342eb0103 <unknown>\nE       #8 0x560342e811ba <unknown>\nE       #9 0x560342e81b8e <unknown>\nE       #10 0x56034311859b <unknown>\nE       #11 0x56034311c34b <unknown>\nE       #12 0x560343104681 <unknown>\nE       #13 0x56034311ceb2 <unknown>\nE       #14 0x5603430e96af <unknown>\nE       #15 0x560343143428 <unknown>\nE       #16 0x560343143600 <unknown>\nE       #17 0x560343152f84 <unknown>\nE       #18 0x7f2b6d494ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1714094081908,"stop":1714094092250,"duration":10342},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560343153e33 <unknown>\n#1 0x560342e43567 <unknown>\n#2 0x560342e8e27d <unknown>\n#3 0x560342e8e331 <unknown>\n#4 0x560342ed1624 <unknown>\n#5 0x560342eb038d <unknown>\n#6 0x560342eceae2 <unknown>\n#7 0x560342eb0103 <unknown>\n#8 0x560342e811ba <unknown>\n#9 0x560342e81b8e <unknown>\n#10 0x56034311859b <unknown>\n#11 0x56034311c34b <unknown>\n#12 0x560343104681 <unknown>\n#13 0x56034311ceb2 <unknown>\n#14 0x5603430e96af <unknown>\n#15 0x560343143428 <unknown>\n#16 0x560343143600 <unknown>\n#17 0x560343152f84 <unknown>\n#18 0x7f2b6d494ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"public ip address","time":{"start":1714094092260,"stop":1714094092302,"duration":42},"status":"passed","steps":[],"attachments":[{"uid":"1daa5c60486847fa","name":"HTTP logs of http://checkip.amazonaws.com/","source":"1daa5c60486847fa.txt","type":"text/plain","size":357}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"a30d6350b397064e","name":"Cookies","source":"a30d6350b397064e.json","type":"application/json","size":3638},{"uid":"401a50a704914bf3","name":"Session Storage","source":"401a50a704914bf3.json","type":"application/json","size":359},{"uid":"765719f164b9987a","name":"Local Storage","source":"765719f164b9987a.json","type":"application/json","size":350},{"uid":"667efb81e261bf94","name":"Console Logs","source":"667efb81e261bf94.json","type":"application/json","size":7289},{"uid":"cc3619f6fda656e5","name":"Full Page Screenshot","source":"cc3619f6fda656e5.png","type":"image/png","size":6246649},{"uid":"e432305bd8e2a5bf","name":"URL","source":"e432305bd8e2a5bf.uri","type":"text/uri-list","size":23},{"uid":"6cb99f8318c3cfcc","name":"Network Logs","source":"6cb99f8318c3cfcc.json","type":"application/json","size":844171}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1714094147717,"stop":1714094147717,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"story","value":"Forgot Password Feature's Functionality"},{"name":"subSuite","value":"Custom sub suite"},{"name":"suite","value":"Custom suite"},{"name":"epic","value":"Security"},{"name":"parentSuite","value":"Custom parent suite"},{"name":"severity","value":"critical"},{"name":"tag","value":"security"},{"name":"host","value":"fv-az1118-655"},{"name":"thread","value":"2478-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.forgot_password_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"TESTCASE-2","url":"https://github.com/allure-examples/allure-examples/issues/2","type":"tms"},{"name":"ISSUE-1","url":"https://github.com/allure-examples/allure-examples/issues/1","type":"issue"},{"name":"Allure Examples","url":"https://github.com/allure-examples/","type":"link"}],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security"]},"source":"56725564af960f94.json","parameterValues":[]}